netci.groovy

```diff
@@ -1742,19 +1742,23 @@ combinedScenarios.each { scenario ->
                                         break
                                     }
                                     else {
+                                        // Setup variables to hold emulator folder path and the rootfs mount path
+                                        buildCommands += "armemul_path=/opt/linux-arm-emulator ; armrootfs_mountpath=/opt/linux-arm-emulator-root"
                                         // Unmount previously mounted rootfs and mount the Linux ARM emulator rootfs at /opt/linux-arm-emulator-root/
-                                        buildCommands += "if grep -qs '/opt/linux-arm-emulator-root' /proc/mounts; then sudo umount /opt/linux-arm-emulator-root; fi ; sudo mount /opt/linux-arm-emulator/platform/rootfs-t30.ext4 /opt/linux-arm-emulator-root/"
-                                        // Remove old copy of coreclr and copy the latest version of coreclr
-                                        // This need to be done as it is not possible to clone the repository inside the chroot jail
-                                        buildCommands += "sudo rm -rf /opt/linux-arm-emulator-root/home/coreclr; sudo mkdir /opt/linux-arm-emulator-root/home/coreclr; sudo cp -R ./ /opt/linux-arm-emulator-root/home/coreclr"
-
-                                        // Chroot into the Linux ARM emulator environment and execute the build
-                                        buildCommands += """echo \"Chrooting into Linux ARM emulator environment\"
-                                                            sudo chroot /opt/linux-arm-emulator-root/ /bin/bash -x <<EOF
-                                                            source /dotnet/setenv/setenv_coreclr.sh
-                                                            cd home/coreclr
-                                                            ./build.sh arm clean verbose skipmscorlib
```

Comment:
@prajwal-aithal have you confirmed that this PR does what you expect it to? This line is causing "dotnet-bot test CI please' to fail every time, as you don't escape the '$' before armrootfs_mountpath, so the groovy compiler thinks you're trying to use a local variable named armrootfs_mountpath, when no such variable exists in netci.groovy. I suspect this won't work as you want it to anyways, since I believe each command is executed in a new shell in the CI, so any variable you initialize in a shell in one command (i.e. armrootfs_mountpath) won't be available when the next command executes. I could be wrong about this, however - if everything is working how you expect after this PR went in, then we need to find some other solution to the 'dotnet-bot test CI please' problem - we can't afford to have that job fail every time, otherwise there's no way to know if a change to netci.groovy is going to break our CI.